data_migrator.py

```diff
@@ -0,0 +1,101 @@
+    etl.todb(poc_dummy_address, to_db_conn, 'poc')
+
+    project_names_table = etl.fromdb(from_db_conn, "select distinct project_name from project")
+    moc_project_transformed = etl.addrownumbers(project_names_table)
+    moc_project_transformed = etl.rename(moc_project_transformed, {'row': 'moc_project_id'})
+    etl.todb(moc_project_transformed, to_db_conn, 'moc_project')
+
+    domain = etl.fromdb(from_db_conn, "select * from domain")
+    domain_table_transformed = etl.cutout(domain, 'domain_uid')
+    domain_table_transformed = etl.rename(domain_table_transformed, {'domain_id': 'service_id', 'domain_name': 'service_name'})
+    etl.todb(domain_table_transformed, to_db_conn, 'service')
+
+    project = etl.fromdb(from_db_conn, "select * from project")
+    moc_project = etl.fromdb(to_db_conn, "select * from moc_project")
+    project_moc_project_joined = etl.join(project, moc_project, key='project_name')
+    project_table_transformed = etl.cutout(project_moc_project_joined, 'project_name')
+    project_table_transformed = etl.rename(project_table_transformed, {'domain_id': 'service_id', 'project_uid': 'project_uuid'})
+    etl.todb(project_table_transformed, to_db_conn, 'project')
+
+    institution2project = etl.fromdb(from_db_conn, "Select * from institution2project")
+    project = etl.fromdb(to_db_conn, "select project_id, moc_project_id from project")
+    inst2project_project_joined = etl.join(institution2project, project, key='project_id')
+    inst2moc_project = etl.cutout(inst2project_project_joined, 'domain_id')
+    etl.todb(inst2moc_project, to_db_conn, 'institution2moc_project')
+
+    project2poc = etl.fromdb(from_db_conn, "select * from project2poc")
+    project2poc_project_joined = etl.join(project2poc, project, key='project_id')
+    poc2moc_project = etl.cutout(project2poc_project_joined, 'project_id', 'domain_id')
+    poc2moc_project = etl.addfield(poc2moc_project, 'role_id', 1)
+    poc2moc_project = etl.addfield(poc2moc_project, 'poc_poc_id', lambda rec: rec['poc_id'])
+    etl.todb(poc2moc_project, to_db_conn, 'poc2moc_project')
+
+    poc2project = etl.cutout(project2poc, 'domain_id')
+    poc2project = etl.addfield(poc2project, 'role_id', 1)
+    poc2project = etl.addfield(poc2project, 'username', '')
+    etl.todb(poc2project,to_db_conn, 'poc2project')
+
+    item = etl.fromdb(from_db_conn, "select * from item")
+    item_transformed = etl.cutout(item, 'domain_id')
+    etl.todb(item_transformed, to_db_conn, 'item')
+
+    raw_item_ts_unique = etl.fromdb(from_db_conn, "WITH summary AS ( SELECT its.item_id, its.start_ts, its.end_ts, its.state, its.catalog_item_id, ROW_NUMBER() OVER(PARTITION BY its.item_id) AS rk FROM ITEM_TS its) SELECT s.* FROM summary s WHERE s.rk = 1")
+    raw_item_ts_unique = etl.cutout(raw_item_ts_unique, 'rk')
+    etl.todb(raw_item_ts_unique, to_db_conn, 'raw_item_ts')
+
+
+if __name__ == '__main__':
+    source_db_config = {"dbname":"", "host":"", "user": "", "pass": ""}
```

Comment:
Put a comment here explaining that this object represents the config/auth to the older Database and needs to be filled in once before running the code.

Reply:
Added the comment.